src/components/history-and-comments.component.tsx

```diff
@@ -4,15 +4,15 @@ import { OverflowMenuVertical } from "@carbon/react/icons";
 import { useTranslation } from "react-i18next";
 import { parseDate, formatDatetime } from "@openmrs/esm-framework";
 import styles from "./history-and-comments.scss";
-import { useOrderDetails } from "../medication-request/medication-request.resource";
-import MedicationCard from "./medication-card.component";
+import { usePrescriptionDetails } from "../medication-request/medication-request.resource";
+import MedicationEventCard from "./medication-event-card.component";
```

Comment:
Just did some re-naming here that hopefully makes things clearer.... also I created a new car that *just* displays a Medication, so I needed to rename MedicationCard